src/functions-and-arrays.js

```diff
@@ -52,14 +90,37 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+  if (arr.length === 0){
+    return null
+  }
+  let newArray = [];
+  for (let i = 0; i < arr.length; i++){
+    if (newArray.indexOf(arr[i]) == -1) {
+      newArray.push(arr[i])
+    }
+  }
+    return newArray
+}
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(array,value) {
+  if (array.length === 0){
+    return null
+  }
+  for (let i = 0; i < array.length; i++) {
+   if (array.indexOf(value) != -1) {
+      return true
+    }
+    else if (array.indexOf(value) == -1) {
```

Comment:
You return false already in the first iteration of your loop. You should only return false after the loop is finished.